src/main/java/com/spring/rest/controller/UserController.java

```diff
@@ -48,5 +48,11 @@ public User getUser(@PathVariable String uname) throws UserNotFoundException {
 		
 		return ResponseEntity.status(HttpStatus.CREATED).build();
 	}
+	@RequestMapping(value="/User/{uname}", method = RequestMethod.DELETE)
+	public ResponseEntity <String> deleteUser(@PathVariable String uname) {
+		try {userService.deleteUser(uname);}catch(UserNotFoundException bex) {};
+		
+		return ResponseEntity.status(HttpStatus.CREATED).build();

```

Comment:
Why the status code here is **HttpStatus.CREATED**? I guess this will generate 201. In case of successful delete, it should be 204 No Content